website/cue/reference/components/transforms/route.cue

```diff
@@ -66,6 +66,15 @@ components: transforms: route: {
 		}
 	}
 
+	example_config: {
+		common: {
+			route: {
+				success: #".status >= 200 && .status < 400"#
+				failure: #".status >= 400"#
```

Comment:
Minor nit, but this wouldn't work, because `.status` can be a non-integer type. Wrapping them in `int!()` would work.

Reply:
Good catch